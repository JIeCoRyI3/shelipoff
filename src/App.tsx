import React, { FC } from 'react';
import HeroSection from 'sections/HeroSection/HeroSection';

import * as Styled from './styles';

const App: FC = () => {
	return (
		<Styled.App>
			<HeroSection />
		</Styled.App>
	);
};

export default App;
