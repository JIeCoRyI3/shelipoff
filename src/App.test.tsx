import '@testing-library/jest-dom';

import { render } from '@testing-library/react';
import React from 'react';

import App from './App';

Object.defineProperty(HTMLMediaElement.prototype, 'muted', {
	set: () => {},
});

describe('App Component', function () {
	it('should match snapshot', async function () {
		const container = render(<App />);
		expect(container).toMatchSnapshot();
	});
});
