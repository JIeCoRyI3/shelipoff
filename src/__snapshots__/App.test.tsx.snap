// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c4 {
  width: 100vw;
  height: 100vh;
  object-fit: cover;
}

.c6 {
  position: absolute;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  font-size: 100px;
  font-family: 'Roboto Mono',monospace;
  font-weight: 400;
  text-transform: uppercase;
  color: white;
  z-index: 3;
}

.c2 {
  position: fixed;
  z-index: 2;
  top: 0;
  left: 0;
}

.c3 {
  position: relative;
  height: 100vh;
  width: 100vw;
}

.c1 {
  width: 100vw;
  height: 100vh;
}

.c5 {
  font-size: 30px;
  color: white;
  font-family: 'Roboto Mono',monospace;
  font-weight: bold;
  text-transform: uppercase;
  position: absolute;
  width: 180px;
  height: 180px;
  cursor: pointer;
  background-color: rgba(0,0,0,0.2);
  top: 50%;
  left: 50%;
  border-radius: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  z-index: 3;
  -webkit-transition: 0.4s;
  transition: 0.4s;
  -webkit-animation: ripple 2600ms infinite;
  animation: ripple 2600ms infinite;
  border: none;
  -webkit-transform-origin: center;
  -ms-transform-origin: center;
  transform-origin: center;
}

.c5:hover {
  background-color: rgba(0,0,0,0.6);
  -webkit-transform: translate(-50%,-50%) scale(1.2);
  -ms-transform: translate(-50%,-50%) scale(1.2);
  transform: translate(-50%,-50%) scale(1.2);
}

.c0 {
  min-height: 500vh;
}

<body>
    <div>
      <div
        class="c0"
      >
        <section
          class="c1"
        />
        <section
          class="c2"
          tabindex="0"
        >
          <div
            class="c3"
          >
            <video
              autoplay=""
              class="c4"
              loop=""
            >
              <source
                class=""
                src="test-file-stub"
              />
            </video>
            <button
              class="c5"
            >
              Click me
            </button>
            <div
              class="c6"
            />
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": .c4 {
  width: 100vw;
  height: 100vh;
  object-fit: cover;
}

.c6 {
  position: absolute;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  font-size: 100px;
  font-family: 'Roboto Mono',monospace;
  font-weight: 400;
  text-transform: uppercase;
  color: white;
  z-index: 3;
}

.c2 {
  position: fixed;
  z-index: 2;
  top: 0;
  left: 0;
}

.c3 {
  position: relative;
  height: 100vh;
  width: 100vw;
}

.c1 {
  width: 100vw;
  height: 100vh;
}

.c5 {
  font-size: 30px;
  color: white;
  font-family: 'Roboto Mono',monospace;
  font-weight: bold;
  text-transform: uppercase;
  position: absolute;
  width: 180px;
  height: 180px;
  cursor: pointer;
  background-color: rgba(0,0,0,0.2);
  top: 50%;
  left: 50%;
  border-radius: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  z-index: 3;
  -webkit-transition: 0.4s;
  transition: 0.4s;
  -webkit-animation: ripple 2600ms infinite;
  animation: ripple 2600ms infinite;
  border: none;
  -webkit-transform-origin: center;
  -ms-transform-origin: center;
  transform-origin: center;
}

.c5:hover {
  background-color: rgba(0,0,0,0.6);
  -webkit-transform: translate(-50%,-50%) scale(1.2);
  -ms-transform: translate(-50%,-50%) scale(1.2);
  transform: translate(-50%,-50%) scale(1.2);
}

.c0 {
  min-height: 500vh;
}

<div>
    <div
      class="c0"
    >
      <section
        class="c1"
      />
      <section
        class="c2"
        tabindex="0"
      >
        <div
          class="c3"
        >
          <video
            autoplay=""
            class="c4"
            loop=""
          >
            <source
              class=""
              src="test-file-stub"
            />
          </video>
          <button
            class="c5"
          >
            Click me
          </button>
          <div
            class="c6"
          />
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
